documentation/staging/content/samples/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -116,19 +116,25 @@ Now that you have created the AKS cluster, installed the operator, and verified
 
 ##### Create secrets
 
-You will use the `kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh` script to create the domain credentials as a Kubernetes secret. Please run:
+You will use the `kubernetes/samples/scripts/create-weblogic-domain-credentials/create-weblogic-credentials.sh` script to create the domain WebLogic administrator credentials as a Kubernetes secret. Please run:
 
 ```
 # cd kubernetes/samples/scripts/create-weblogic-domain-credentials
 ```
 ```shell
-$ ./create-weblogic-credentials.sh -u weblogic -p welcome1 -d domain1
+$ ./create-weblogic-credentials.sh -u <a username> -p <a password> -d domain1
```

Comment:
<a username> and <a password> looks a little odd on first glance,  further down  is using <wl admin username> <wl admin password>,  can we make it consistent?